src/cdk/testing/tests/protractor.e2e.spec.ts

```diff
@@ -200,11 +200,15 @@ describe('ProtractorHarnessEnvironment', () => {
 
     it('should be able to clear', async () => {
       const input = await harness.input();
+      const inputEvent = await harness.inputEvent();
       await input.sendKeys('Yi');
       expect(await input.getProperty('value')).toBe('Yi');
+      expect(await inputEvent.text()).toBe('Count: 2');
 
       await input.clear();
       expect(await input.getProperty('value')).toBe('');
+      // Doesn't change input event count on clear. Should it?
```

Comment:
Hmm, that seems like a bug. It's probably because nothing is triggering change detection. The component is probably set up to run change detection on keydown events, but the clear method isn't sending a keydown event. Maybe we could change it to send a backspace or something

Reply:
> The component is probably set up to run change detection on keydown events
The component is marked for `OnPush` change detection.
I am using `Observable`s and `async` pipe, so it should have updated?

Reply:
Ah now I understand your commment.
Updated `protractor-element.ts` to send a backspace before clear (sending backspace after clear doesn't seem to help).